scripts/ObstacleDetector2D.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
```

Comment:
It's worth putting python2 here, since our ROS stuff explicitly runs on python2. Should be fine, though.